recipe/meta.yaml

```diff
@@ -25,6 +30,7 @@ requirements:
     - cmake
     - make                               # [unix]
     - pkg-config                         # [unix]
+    - patch                              # [not win]
```

Comment:
Why is patch required?
```
  patches:
    - CMakeLists.patch  # [win and vc<14]
```

Also note that `patches:` is a dangling entry for non-win. It would be cleaner if it were disabled as well.

Reply:
Since we entirely skip windows, maybe it would be better to comment out the patch section and  add a note regarding the patch?

Reply:
Dropped the patch, wasn't applicable any which way. Thanks!